<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageWidget</class>
 <widget class="QWidget" name="ImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="toolSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="plotSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="upperPlotWidget" native="true">
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QSplitter" name="upperPlotSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="twoDWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>100</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="0">
            <property name="margin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="twoDVerticalLayout"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="oneDRightWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="margin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="oneDRightHorizontalLayout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="lowerPlotWidget" native="true">
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QSplitter" name="lowerPlotSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="oneDBottomWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_3" columnstretch="0">
            <property name="margin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="oneDBottomVerticalLayout"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="cornerWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="margin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="cornerHorizontalLayout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="toolWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="toolVerticalLayout"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="infoLabel">
           <property name="toolTip">
            <string>coordinate info display for the mouse pointer</string>
           </property>
           <property name="text">
            <string>[x1, y1, x2, y2], sum: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="infoLineEdit">
           <property name="toolTip">
            <string>coordinate info display for the mouse pointer</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="toolComboBox">
           <property name="toolTip">
            <string>select the image tool for the mouse pointer, i.e. Intensity, ROI, LineCut, Angle/Q</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>toolSplitter</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>twoDWidget</zorder>
  <zorder>oneDBottomWidget</zorder>
  <zorder>oneDRightWidget</zorder>
  <zorder>cornerWidget</zorder>
  <zorder></zorder>
  <zorder>upperPlotWidget</zorder>
  <zorder>lowerPlotWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
